frontend/src/pages/home/new-todo/new-todo.jsx

```diff
@@ -0,0 +1,33 @@
+import { createTodo } from '../../../api/api.js';
+import Modal from '../../../modal/ModalTemplate.jsx';
+import './new-todo.css'
+
+function NewTodoButton(props) {
+  const success = async () => {
+    props.modal(null);
+    const title = document.querySelector('.modal-input').value;
```

Comment:
Do not use `querySelector`; that breaks the purpose of React. If you are unsure how to work around this, let's set up a meeting to discuss this.